1-Draft/RFCXXXX-Send-MailMessage.md

```diff
@@ -0,0 +1,95 @@
+---
+RFC: RFCXXXX
+Author: Travis Plunk
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: cmdlets
+Comments Due: 4/30/2019
+Plan to implement: No
+---
+
+# Obsoleting `Send-MailMessage`
+
+`Send-MailMessage` does not support many modern protocols leading to the inability to use this with modern secure mail services.
```

Comment:
I question this statement. Do we have any numbers to back this up?

Most of the time when I am using Send-MailMessage, I am hitting an internal SMTP server or Office 365 (which doesn't seem to be an issue yet). If the majority of users are hitting these, then this CMDLet has quite a long life left in it.

Reply:
It doesn't matter how many there are. Users need a reasonable expectation that what they're doing is secure if it's reporting as secure. If you're using an internal-only server, we can provide some context so that you can decide it's okay to do what you're doing, but we can't create a false expectation of security.